### DISCUSSION TITLE
Not full css

### DISCUSSION THREAD

**Discussion by SatyaveerPal** [tailwindlabs/tailwindcss#6424] [General]
Running tailwindcss -i src/input.css -o public/css/styles.css only gives me css with around 470 lines of normalize code. Can any help please

**Comment by adamwathan** [comment]
Hey! Tailwind 3 has no way to generate a "full" build, it is JIT-only so it will only ever generate the CSS you are actually using.

If you want to ensure it always includes certain classes, you can use the safelisting feature:

https://tailwindcss.com/docs/content-configuration#safelisting-classes
